src/ast/rewriter/bit_blaster/bit_blaster_tpl_def.h

```diff
@@ -178,175 +157,70 @@ void bit_blaster_tpl<Cfg>::mk_subtracter(unsigned sz, expr * const * a_bits, exp
 
 template<typename Cfg>
 void bit_blaster_tpl<Cfg>::mk_multiplier(unsigned sz, expr * const * a_bits, expr * const * b_bits, expr_ref_vector & out_bits) {
-    SASSERT(sz > 0);
```

Comment:
yes, we tried several things over time.
It is right, we might as well kick this out.

Reply:
Though, for a careful merge it is better to have side-by-side with old version and run QF_BV benchmarks fully to evaluate effect.
I can do this easily on our regression infrastructure.